flake.nix

```diff
@@ -74,8 +74,20 @@
         ${getExe pkgs.reuse} --suppress-deprecation lint
         ${getExe pkgs.lychee} --offline --no-progress .
         ${getExe lint-python}
+        ${getExe format-cpp}
       '';
 
+      format-cpp = pkgs.writeShellApplication {
```

Comment:
It would be nice if this included followed the naming conventions of sonata-software [`lint-cpp`](https://github.com/lowRISC/sonata-software/blob/72e04e7ef29afe0ee3d1833eb0ac4be0bf765981/flake.nix#L143). I'd like to add `clang-tidy` in the future.